pkg/ccl/backupccl/restore_span_covering.go

```diff
@@ -11,245 +11,23 @@ package backupccl
 import (
 	"sort"
 
-	"github.com/cockroachdb/cockroach/pkg/keys"
 	"github.com/cockroachdb/cockroach/pkg/roachpb"
-	"github.com/cockroachdb/cockroach/pkg/sql/covering"
 	"github.com/cockroachdb/cockroach/pkg/sql/execinfrapb"
-	"github.com/cockroachdb/cockroach/pkg/util/hlc"
 	"github.com/cockroachdb/cockroach/pkg/util/interval"
-	"github.com/cockroachdb/errors"
 )
 
 type intervalSpan roachpb.Span
 
 var _ interval.Interface = intervalSpan{}
 
-// ID is part of `interval.Interface` but unused in makeImportSpans.
+// ID is part of `interval.Interface` but unused in bac.
```

Comment:
nit: comment seems truncated, but can cleanup later

Reply:
fixed. I think we can remove the intervalSpan entirely and just use `roachpb.SpanGroup` for `SpansForAllIndexes`, since I _think_ it is just there to accumulate/merge overlapping spans and then return the the final set of disjoint spans, which spqngroup.Add()+Slice would do for us in a lot less code (backed by the same tree I think)